<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Password Generator</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>700</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>700</height>
    </size>
   </property>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>100</y>
      <width>450</width>
      <height>500</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="tabBarAutoHide">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="Generator">
     <attribute name="title">
      <string>Generator</string>
     </attribute>
     <widget class="QLineEdit" name="GeneratedPassword">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>250</y>
        <width>250</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="GeneratePassword">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>250</width>
        <height>100</height>
       </rect>
      </property>
      <property name="text">
       <string>Generate password</string>
      </property>
     </widget>
     <widget class="QPushButton" name="CopyPassword">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>300</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Copy</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ClearPassword">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>300</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Clear</string>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordStrength">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>380</y>
        <width>250</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordStrengthLabel">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>350</y>
        <width>250</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Password strength</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Database">
     <attribute name="title">
      <string>Database</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>75</y>
        <width>400</width>
        <height>250</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Sunken</enum>
      </property>
      <property name="sizeAdjustPolicy">
       <enum>QAbstractScrollArea::AdjustToContents</enum>
      </property>
      <property name="dragEnabled">
       <bool>true</bool>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="textElideMode">
       <enum>Qt::ElideLeft</enum>
      </property>
      <attribute name="horizontalHeaderMinimumSectionSize">
       <number>40</number>
      </attribute>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>190</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
     <widget class="QPushButton" name="CreateDatabase">
      <property name="geometry">
       <rect>
        <x>275</x>
        <y>30</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Create Database</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Save">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>350</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Save</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Delete">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>350</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <widget class="QPushButton" name="Edit">
      <property name="geometry">
       <rect>
        <x>175</x>
        <y>350</y>
        <width>100</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
     <widget class="QPushButton" name="OpenDatabase">
      <property name="geometry">
       <rect>
        <x>25</x>
        <y>30</y>
        <width>150</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Open Database</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="GeneratedPassword_2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>410</y>
        <width>250</width>
        <height>24</height>
       </rect>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Settings">
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLabel" name="UpperCase">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>50</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>UpperCase</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="LowerCase">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>100</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>LowerCase</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Numbers">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>150</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Numbers</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="Symbols">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>200</y>
        <width>100</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Symbols</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
     <widget class="QSpinBox" name="UpperCase_spinBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>50</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="LowerCase_spinBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>100</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Numbers_spinBox">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>150</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QSpinBox" name="Symbols_spinBox">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>50</width>
        <height>25</height>
       </rect>
      </property>
      <property name="minimum">
       <number>1</number>
      </property>
      <property name="maximum">
       <number>8</number>
      </property>
      <property name="value">
       <number>2</number>
      </property>
     </widget>
     <widget class="QSlider" name="HorizontalSlider">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>275</y>
        <width>250</width>
        <height>25</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <strikeout>false</strikeout>
       </font>
      </property>
      <property name="minimum">
       <number>0</number>
      </property>
      <property name="maximum">
       <number>32</number>
      </property>
      <property name="value">
       <number>8</number>
      </property>
      <property name="tracking">
       <bool>false</bool>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="tickPosition">
       <enum>QSlider::NoTicks</enum>
      </property>
     </widget>
     <widget class="QLabel" name="PasswordLength">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>325</y>
        <width>250</width>
        <height>50</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>12</pointsize>
       </font>
      </property>
      <property name="text">
       <string>8</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="textInteractionFlags">
       <set>Qt::LinksAccessibleByMouse</set>
      </property>
     </widget>
     <widget class="QPushButton" name="ResetButton">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>370</y>
        <width>80</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Reset</string>
      </property>
     </widget>
     <widget class="QCheckBox" name="UpperCase_checkBox">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>50</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="LowerCase_checkBox">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>100</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="Numbers_checkBox">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>150</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QCheckBox" name="Symbols_checkBox">
      <property name="geometry">
       <rect>
        <x>325</x>
        <y>200</y>
        <width>25</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="MainTitle">
    <property name="geometry">
     <rect>
      <x>25</x>
      <y>25</y>
      <width>450</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>15</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Password Generator</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
